stl/inc/memory

```diff
@@ -1126,11 +1126,11 @@ private:
 #ifdef _M_CEE_PURE
     // permanent workaround to avoid mentioning _purecall in msvcurt.lib, ptrustu.lib, or other support libs
     virtual void _Destroy() noexcept {
-        _STD terminate();
+        _STD abort();

```

Comment:
We conventionally say `_CSTD abort` instead of `_STD abort`. Although it doesn't matter, I'd like to make the code consistent (i.e. friendlier to searching) when possible. There are some pre-existing occurrences that we can clean up too.